lumigator/python/mzai/evaluator/constants.py

```diff
@@ -0,0 +1,12 @@
+import os
+from pathlib import Path
+
+EVALUATOR_HOME_PATH: str = os.getenv(
+    "EVALUATOR_HOME",
+    str(Path.home() / ".evaluator"),
+)
+
+EVALUATOR_RESULTS_PATH: str = os.getenv(
```

Comment:
In the code there is still LM_BUDDY_RESULTS_PATH (will try to capture those I see here, there might be more around)

Reply:
thank you!